packages/ui/src/components/KafkaClusterStatus/components/ClusterStatusLabel.css

```diff
@@ -0,0 +1,7 @@
+.mas-c-status.pf-l-split.pf-m-gutter > *:not(:last-child) {
```

Comment:
@suyash-naithani this classname is already used somewhere else. We can't reuse the same name, we risk collisions. Please define a different name.

Reply:
Thanks for pointing this out, I've tweaked the name so it is unique